inscriptions/ISic1018.xml

```diff
@@ -170,25 +175,21 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140496">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-
-
-                    𐆠 𐆠
-                    <lb n="2"/>
-                    <supplied reason="lost">— — — ἐ</supplied>νθάδε
-                    <lb n="3"/>
-                    <supplied reason="lost">— — — ζή</supplied>σας ἔτη
-                    <lb n="4"/>
-                    <supplied reason="lost">— — — —</supplied>
-                    καλανδ 𐆠<lb n="5"/>
-                    <surplus>²2 swastikas</surplus>²
-                    <lb n="6"/>
+                    <lb n="0"/><g type="staurogram">&#65952;</g>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost">ἐ</supplied>νθάδε
+                    <lb n="2"/><supplied reason="lost">κεῖται</supplied> <supplied reason="lost">ὁ</supplied> <supplied reason="lost">δεῖνα</supplied> <supplied reason="lost">ζήσ</supplied>ας ἔτη
```

Comment:
Usually before ο δεινος and similar we put `<note>e.g.</note>` to make it explicit. Shall I go over the corpus and have a look at all of them?